ts/interactions/conversationInteractions.ts

```diff
@@ -121,20 +121,48 @@ export async function unblockConvoById(conversationId: string) {
 }
 
 /**
- * marks the conversation as approved.
+ * marks the conversation's approval fields, sends messageRequestResponse, syncs to linked devices
  */
-export const approveConversation = async (conversationId: string) => {
-  const conversationToApprove = getConversationController().get(conversationId);
+export const acceptConversation = async (conversationId: string, syncToDevices: boolean = true) => {
+  const convoToApprove = getConversationController().get(conversationId);
 
-  if (!conversationToApprove || conversationToApprove.isApproved()) {
+  if (!convoToApprove || convoToApprove.isApproved()) {
     window?.log?.info('Conversation is already approved.');
     return;
   }
 
-  await conversationToApprove.setIsApproved(true);
+  Promise.all([
+    await convoToApprove.setIsApproved(true),
+    await convoToApprove.setDidApproveMe(true),
+  ]);
```

Comment:
each of those calls do a commit() to the db. Might be a good idea to have another conversationModel function to update those two fields when needed with only one commit() instead